code/modules/projectiles/ammunition/_ammunition.dm

```diff
@@ -33,12 +34,28 @@
 	///pacifism check for boolet, set to FALSE if bullet is non-lethal
 	var/harmful = TRUE
 
+/// Creates a new projectile in order to retrieve offensive data, adding on relevant stamina or pellet count information as needed
+/obj/item/ammo_casing/proc/get_notes()
+	if(projectile_type)
+		/// Output added to the offensive notes variable
+		var/note_builder = ""
+		var/obj/projectile/exam_proj = new src.projectile_type
+		note_builder = uppertext("[exam_proj.damage_type]: [exam_proj.damage]\n")
+		if(exam_proj.stamina)
+			note_builder += "STAM: [exam_proj.stamina]\n"
+		if(pellets > 1)
+			note_builder += "COUNT: [pellets]\n"
+		return note_builder
+	else
+		return
```

Comment:
we've discussed this on discord, but this is decidedly the wrong way to go about this, partially because initial() exists, and partially because this isn't a very flexible system